src/components/taskDetails/TaskDates.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Details from './Details';
+import styles from './task-details.module.scss';
+import { TASK_EXTENSION_REQUEST_URL } from '@/constants/url';
+import convertTimeStamp from '@/helperFunctions/convertTimeStamp';
+
+interface TaskDatesProps {
+    isEditing: boolean;
+    isUserAuthorized: boolean;
+    startedOn: string;
+    endsOn: number;
+    newEndOnDate: string;
+    setNewEndOnDate: (date: string) => void;
+    handleBlurOfEndsOn: () => void;
+    isExtensionRequestPending: boolean;
+    taskId: string;
+}
+
+const TaskDates: React.FC<TaskDatesProps> = ({
+    isEditing,
+    isUserAuthorized,
+    startedOn,
+    endsOn,
+    newEndOnDate,
+    setNewEndOnDate,
+    handleBlurOfEndsOn,
+    isExtensionRequestPending,
+    taskId,
+}) => {
+    const getExtensionRequestLink = (
+        taskId: string,
+        isExtensionRequestPending?: boolean
+    ) => {
+        return isExtensionRequestPending
+            ? `${TASK_EXTENSION_REQUEST_URL}?&q=${encodeURIComponent(
+                  `taskId:${taskId},status:PENDING`
+              )}`
+            : null;
+    };
+
+    const formattedEndsOn = endsOn ? convertTimeStamp(endsOn) : 'TBD';
+
+    return (
+        <>
+            <div className={styles.inputContainer}>
+                <Details detailType={'Started On'} value={startedOn} />
+            </div>
+            <div className={styles.inputContainer}>
+                <Details
+                    detailType={'Ends On'}
```

Comment:
The `Details` component currently converts `detailType` values like `Started On` and `Ends On` to more user-friendly labels like `Started` and `Ended`. If we change `detailType` to camel case (e.g., `startedOn` and `endsOn`), the `Details` component will no longer automatically format these into `Started` and `Ended`.


To implement this change, I'll need to update the `Details` component to recognize and convert the new camel case values back into the user-friendly labels we're currently displaying. This will involve modifying the logic in `Details` to handle the new `detailType` values appropriately.


- will solve this in the next issue #1246